docker/try_mysql.bat

```diff
@@ -0,0 +1,48 @@
+@echo off

```

Comment:
Same question for the bat / sh files, is it possible to use one single file for both platforms?